### PR TITLE
Update da-DK.txt

### PR DESCRIPTION
**Pull request by LPSGizmo**
Correction made to #2509

Applying for issue(s):
- #2535
- #2550 
- #2551 
- #2558 
- #2562 
- #2572 
- #2578 
- #2594 
- #2602 
- #2609 
- #2620 
- #2625 
- #2631 
- #2641 
- #2650 
- #2658 
- #2659
- #2678  
- #2679 

EDIT (by @tupaschoal):
Additional issues applied:

- #2327
- #2345
- #2363
- #2509
- #2563
- Partial #1979?

### CODE REVIEW

File: data/language/da-DK.txt [hunk 1]
```diff
@@ -3625,14 +3630,73 @@ STR_6525    :Genindlæs alle aktiver i spillet med de aktiverede materiel pakker
 STR_6526    :(basis grafik, musik og lydeffekter)
 STR_6527    :Konkurrencer
 STR_6528    :Ugyldig spor parameter!
-STR_6529    :Invalid farve skema parameter!
-STR_6530    :User Created Expansion Set
+STR_6529    :Ugyldig farve skema parameter!
+STR_6530    :Brugergenereret udvidelser
```
Comment (tupaschoal) [inline data/language/da-DK.txt:3634]:
```suggestion
STR_6530    :User Created Expansion Set
```
Like you said, this line should not be translated.

File: data/language/da-DK.txt [hunk 2]
```diff
@@ -178,6 +179,7 @@ STR_0600    :Selvkørende togvogne kører igennem en sen snoet bane af spor
 STR_0602    :Rutsjebanetog accelereres ud af stationen af lineære induktionsmotorer for at suse gennem snoede inversioner
 STR_0603    :En rutsjebane i træ stil, med stål spor, Med stejle fald og hurtige sving.
 STR_0604    :Gæster sider i en enkelt række, på et smalt monorail spor, imens de suser igennem skarpe sving ,snoninger og hurtige vridninger.
+STR_0605    :Riders toboggan down a meandering steel track, braking to control their speed
```
Comment (tupaschoal) [inline data/language/da-DK.txt:182]:
```suggestion
STR_0605    :Ryttere kælker ned ad en bugtende stålbane og bremser for at kontrollere deres hastighed
```

File: data/language/da-DK.txt [hunk 3]
```diff
@@ -3625,14 +3630,73 @@ STR_6525    :Genindlæs alle aktiver i spillet med de aktiverede materiel pakker
 STR_6526    :(basis grafik, musik og lydeffekter)
 STR_6527    :Konkurrencer
 STR_6528    :Ugyldig spor parameter!
-STR_6529    :Invalid farve skema parameter!
-STR_6530    :User Created Expansion Set
+STR_6529    :Ugyldig farve skema parameter!
+STR_6530    :Brugergenereret udvidelser
 STR_6531    :The Time Machine
 STR_6532    :Katy’s Dreamworld
```
Comment (tupaschoal) [inline data/language/da-DK.txt:3636]: Missing translation for these two?

File: data/language/da-DK.txt [hunk 4]
```diff
@@ -1958,7 +1960,7 @@ STR_2749    :Mit nye scenarie
 STR_2750    :Flyt alle enheder til toppen
 STR_2751    :Flyt alle enheder til bunden
 STR_2752    :Ryd græs
-STR_2753    :Slået græs
+STR_2753    :Slå græs
```
Comment (tupaschoal) [inline data/language/da-DK.txt:1963]: I see this change comes from #1979, but it doesn't seem like you reviewed the other changes there, can you please check?

File: data/language/da-DK.txt (hunk 3 above)
Comment (LPSGizmo) [reply data/language/da-DK.txt:3636]: Should they be translated?

If "translation needed" then 
{
STR_6531    :Tidsmaskinen
STR_6532    :Katy’s Drømmeverden
}
Endif

:)

File: data/language/da-DK.txt (hunk 4 above)
Comment (LPSGizmo) [reply data/language/da-DK.txt:1963]: this is a language "fix" from danish (done cutting the grass) to (cut the grass).

File: data/language/da-DK.txt (hunk 1 above)
Comment (LPSGizmo) [reply data/language/da-DK.txt:3634]: STR_6530    :User Created Expansion Set

is the correct "translation"

File: data/language/da-DK.txt [hunk 5]
```diff
@@ -3625,14 +3630,73 @@ STR_6525    :Genindlæs alle aktiver i spillet med de aktiverede materiel pakker
...
+STR_6565    :Havskum grøn
+STR_6566    :Violet
+STR_6567    :Lavendel
+STR_6568    :Pastel orange
+STR_6569    :Dybt vand
+STR_6570    :Pastel pink
+STR_6571    :Umber
+STR_6572    :Beige
+STR_6573    :Usynlig
+STR_6574    :Void
```
[the remark this answers is not in the document]
Comment (LPSGizmo) [reply data/language/da-DK.txt:3679]: Some could be translated. Rest is ok.

STR_6553    :Solbrun
STR_6554    :Rødbrun
STR_6555    :Koral rosa
STR_6557    :Chartreuse
STR_6559    :Celadon
STR_6566    :Violet
STR_6568    :Pastel orange
STR_6570    :Pastel pink
STR_6571    :Umber
STR_6572    :Beige
STR_6574    :Tomrum

**Comment by LPSGizmo** [comment]
should I fix the missing/wrong translations, and the create a new pull request?

File: data/language/da-DK.txt (hunk 4 above)
Comment (LPSGizmo) [reply data/language/da-DK.txt:1963]: Where is this translation used?

File: data/language/da-DK.txt (hunk 3 above)
Comment (Gymnasiast) [reply data/language/da-DK.txt:3636]: They are names and don’t need translation.

**Comment by Gymnasiast** [comment]
You don’t need to create a new one necessarily, you can just update this branch if you know how.

Alternatively, you can create smaller ones that only address a few issues each. But that is up to you.

File: data/language/da-DK.txt (hunk 4 above)
Comment (tupaschoal) [reply data/language/da-DK.txt:1963]: It's on the cheats window, it's a button you can click to immediately "mow all grass"

File: data/language/da-DK.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (tupaschoal) [reply data/language/da-DK.txt:3634]:
```suggestion
STR_6530    :User Created Expansion Set
```

File: data/language/da-DK.txt (hunk 5 above)
Comment (tupaschoal) [inline data/language/da-DK.txt:3679]:
```suggestion
STR_6553    :Solbrun
STR_6554    :Rødbrun
STR_6555    :Koral rosa
STR_6556    :Skov grøn
STR_6557    :Chartreuse
STR_6558    :Jæger grøn
STR_6559    :Celadon
STR_6560    :Limegrøn
STR_6561    :Sepia
STR_6562    :Fersken
STR_6563    :Singrøn
STR_6564    :Blågrøn
STR_6565    :Havskum grøn
STR_6566    :Violet
STR_6567    :Lavendel
STR_6568    :Pastel orange
STR_6569    :Dybt vand
STR_6570    :Pastel pink
STR_6571    :Umber
STR_6572    :Beige
STR_6573    :Usynlig
STR_6574    :Tomrum
```

**Comment by tupaschoal** [comment]
@LPSGizmo I've applied the suggestions I made you agreed with and the suggestions you did, if no change is needed on STR_2753 after my comment, then I think we can go ahead and merge this :)

**Pull request merged**